src/test/java/net/snowflake/ingest/streaming/internal/ParameterProviderTest.java

```diff
@@ -310,15 +310,15 @@ public void testValidCompressionAlgorithmsAndWithUppercaseLowerCase() {
         });
     List<String> zstdValues = Arrays.asList("ZSTD", "zstd", "Zstd", "zStd");
```

Comment:
unrelated unhelpful question but I don't understand the need for this test. 
Do we not just normalize the zstdValue input? Why do we only test 4 variant cases of ZSTD? There's ZStd, zSTd, etc